File: Cpp/Math/Sum of Natural Numbers.cpp
//https://www.geeksforgeeks.org/problems/sum-of-series2811/1
class Solution {
  public:
    int seriesSum(int n) {
        // code here
        long sum=0;
        while(n>0){
            sum=n+sum;
            n--;
        }
        return sum;
    }
};

File: Python/Math/Sum of Natural Numbers.py
//https://www.geeksforgeeks.org/problems/sum-of-series2811/1
class Solution:
    def seriesSum(self, n : int) -> int:
        # code here
        sum = 0
        while n > 0:
            sum += n
            n -= 1
        return sum
